src/github.com/matrix-org/dendrite/clientapi/common/eventcontent.go

```diff
@@ -0,0 +1,62 @@
+package common
+
+// CreateContent is the event content for http://matrix.org/docs/spec/client_server/r0.2.0.html#m-room-create
+type CreateContent struct {
```

Comment:
FYI I didn't use `gomatrixserverlib`'s version because it included a bunch of other crap required for auth.  https://github.com/matrix-org/gomatrixserverlib/blob/master/eventcontent.go#L27